uaclient/config.py

```diff
@@ -86,7 +86,7 @@ class UAConfig:
     data_paths = {
         "instance-id": DataPath("instance-id", True, False),
         "machine-access-cis": DataPath("machine-access-cis.json", True, False),
-        "lock": DataPath("lock", True, False),
+        "lock": DataPath("lock", False, False),
```

Comment:
Confirmed behavior on system with boh auto-attach in progress and/or manual ua attach/detach.

This visibility of the lock file also ensures `ua status --wait` blocks for non-root user until operation is complete.